packages/account/src/Components/inline-note-with-icon/inline-note-with-icon.tsx

```diff
@@ -1,8 +1,15 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Icon, Text } from '@deriv/components';
 
-const InlineNoteWithIcon = ({ icon, message, title }) => {
+type TInlineNoteWithIconExtend = {
+    icon: string;
+    message: string;
```

Comment:
Can't this message and string copme with <Localize/>  component? Then it is of type ReactNode. right?